Top32/Top32_1000_aaChannel/Top32_1000_aaChannel_run_card.dat

```diff
@@ -0,0 +1,280 @@
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set                                     
+ 263400    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+ F        = fixed_ren_scale  ! if .true. use fixed ren scale
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.1880  = scale            ! fixed ren scale
+ 91.1880  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.1880  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 0        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
+ 1        = highestmult      ! for ickkw=2, highest mult group
+ 1        = ktscheme         ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+ 1        = alpsfact         ! scale factor for QCD emission vx
+ F        = chcluster        ! cluster only according to channel diag
+ T        = pdfwgt           ! for ickkw=1, perform pdf reweighting
+ 5        = asrwgtflavor     ! highest quark flavor for a_s reweight
+ T        = clusinfo         ! include clustering tag in output
+ 3.0      = lhe_version       ! Change the way clustering information pass to shower.        
+#*********************************************************************
+#**********************************************************
+#
+#**********************************************************
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+   T  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
+#**********************************************************
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15  = bwcutoff      ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#**********************************************************
+   T  = cut_decays    ! Cut decay products 
```

Comment:
Hi Josh,

Thanks again.
I have updated the files, and here is the latest one,
https://github.com/ymtzeng/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/13TeV/Top32/Top32_1000_aaChannel
.

Could you help me to check them again?

Cheers,
Jacky

On Sun, Apr 19, 2015 at 2:31 AM, Josh Bendavid <notifications@github.com>
wrote:

> In Top32/Top32_1000_aaChannel/Top32_1000_aaChannel_run_card.dat
> <https://github.com/cms-sw/genproductions/pull/323#discussion_r28646268>:
>
> > +#**********************************************************
> > +# Automatic ptj and mjj cuts if xqcut > 0
> > +# (turn off for VBF and single top processes)
> > +#**********************************************************
> > +   T  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
> > +#**********************************************************
> > +#
> > +#**********************************
> > +# BW cutoff (M+/-bwcutoff*Gamma)
> > +#**********************************
> > +  15  = bwcutoff      ! (M+/-bwcutoff*Gamma)
> > +#**********************************************************
> > +# Apply pt/E/eta/dr/mij cuts on decay products or not
> > +# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
> > +#**********************************************************
> > +   T  = cut_decays    ! Cut decay products
>
> Yes, this affects the gridpacks, so they would need to be redone if
> changing this. (This is why the recommended procedure is to make pull
> requests for cards before producing the gridpacks so that feedback on the
> cards can be taken into account.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/genproductions/pull/323/files#r28646268>.
>